mkosi

```diff
@@ -1945,8 +1956,11 @@ def unlink_try_hard(path):
 
 def empty_directory(path):
 
-    for f in os.listdir(path):
-        unlink_try_hard(os.path.join(path, f))
+    try:
+        for f in os.listdir(path):
+            unlink_try_hard(os.path.join(path, f))
+    except:
```

Comment:
Also, `except:` should almost never be used. It's a Python gotcha, soemthing that seems natural, but is the wrong thing to do in almost all circumstances, because it catches InterruptExcetion, so ^C doesn't work. If you want to catch "everything", `except Exception` is the way to go. (The exception to this rule is interactive shells, where you **need** to catch ^C, and return to the prompt.)